src/hooks/useClickAway/index.ts

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useRef } from 'react';
+import { TCallback } from './types';
+
+const events = ['mousedown', 'touchstart'];
+
+const useClickAway = (callback: TCallback) => {
+  const ref = useRef(null);
```

Comment:
```suggestion
  const ref = useRef<HTMLElement>(null);
```
이건 어떠신가요? 그렇다면 15Line도
`const element = ref.current `이렇게 수정할 수 있을 것 같습니다!

Reply:
오 어떻게 as 줄일 지 고민이였는데 정은님 덕분에 한 수 배워갑니다 감사해여